scripts/variantstore/AOU_DELIVERABLES.md

```diff
@@ -25,7 +25,7 @@
 - **NOTE** If you want to create a large sample set after you have run the notebook, Terra provides (and recommends you use) this python [script](https://github.com/broadinstitute/firecloud-tools/tree/master/scripts/import_large_tsv) which allows you to upload a sample set to the workspace.
 - **TBD This is based on VCF output; VDS output might need different increases.** For extracting VCFs as the final output for the callset, you will want to increase the Google quotas for the workspace project (you can find this in the workspace dashboard under Cloud Information > Google Project ID) to these levels (all in the workspace region):
   - Persistent Disk Standard (GB): 1,000,000 GB (1 PB)
-  - CPUs: 64,000
+  - CPUs: 6,400
```

Comment:
From poking around the history it looks like this has always been 64,000, though never having done a mega callset I defer to you as to which is correct. 🙂

Reply:
iiiinteresting

Reply:
Fun fact: I requested 64,000 VM instances for the intermediate callset but actually got 640,000 (!)

<img width="1062" alt="Screen Shot 2022-09-09 at 11 18 41 AM" src="https://user-images.githubusercontent.com/10790523/189437645-df732aff-acf5-4326-8602-f10ce9f137dc.png">

Reply:
🤔 maybe that's the reason Google started cracking down....

Reply:
Wow!